frontend/next.config.js

```diff
@@ -10,6 +10,7 @@ const PLAUSIBLE_URL = "https://plausible.io";
 const SCRIPT_SRC = ["'self'", PLAUSIBLE_URL];
 
 module.exports = {
+  eslint: { dirs: ["doc-site", "pages", "components", "lib"] },
```

Comment:
@seve do you remember why you add this? Just realized that we're missing lots of lints in other folders 😆 

Thanks!